website/docs/discord/01-getting-started.mdx

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1.1
+---
+
+import SbtDependency from "@site/src/components/SbtDependency";
+
+# Getting Started
+
+## Installation
+
+To use `ChatOps4s` with Discord, install the following dependency
+
+<SbtDependency moduleName={"chatops4s-discord"} />
+
+## Picking an `sttp` Backend
+
+You must install an `sttp` backend which `ChatOps4s` will use to make HTTP requests on your behalf. View your choices of backends on `sttp`'s [documentation](https://sttp.softwaremill.com/en/latest/backends/summary.html).
+
+## Picking a `tapir` Server Interpreter
+
+You must install a `tapir` server interpreter to allow `ChatOps4s` to receive HTTP requests from Discord. View your choices of server interpreters on `tapir`'s [documentation](https://tapir.softwaremill.com/en/latest/server/overview.html).
+
+## Setting up your Server
+
+Once you have picked a server interpreter and `sttp` backend, import the `Server` and `DiscordInbound` class to set up your sever. In the following example, we create a `ChatOps4s` server using the `http4s` interpreter (using `blaze`) and `cats` backend for `sttp`:
+
+```scala
```

Comment:
Out the the snipper in the examples project (or reuse existing) and import it from there. This is the pattern use for all the snippets in other projects. This was the snippet is always guaranteed to compile.